lib/endpoints/class-wp-json-controller.php

```diff
@@ -103,4 +103,21 @@ public function get_item_schema() {
 		return array();
 	}
 
+	/**
+	 * Get an array of endpoint arguments from the item schema for the controller.
+	 * 
+	 * @return array
+	 */
+	public function get_endpoint_args_for_item_schema() {
+
+		$schema                = $this->get_item_schema();
+		$post_type_fields      = ! empty( $schema['properties'] ) ? $schema['properties'] : array();
+		$post_type_fields_args = array();
+
+		foreach ( $post_type_fields as $field_id => $params ) {
+			$post_type_fields_args[$field_id] = array();
```

Comment:
> I don't understand what you are referring to. Validation works off the args param - we are now taking that from the schema; which prior to this PR was specifying the args incorrectly.

Right — so there should've been some tests failing because of this break.